components/table/test/table.vdiff.js

```diff
@@ -972,6 +972,11 @@ describe('table', () => {
 	});
 
 	describe('mutations', () => {
+		async function removeSelection(elem, selector) {
```

Comment:
I'm curious why this was necessary since I'd expect `_recentTableChange` to be `false` and therefore for the code to behave as it did previously?

Reply:
Now the mutation is the second update in under 300ms, which means it's getting throttled/debounced.

Reply:
Right, I understand. Hopefully that doesn't break other folks' tests in a similar way.

Also now all these tests are `300ms` slower haha. I guess another option would be to use Sinon's fake timers and manually advance the clock `300ms` before calling `remove()`.